mlflow/langchain/__init__.py

```diff
@@ -247,7 +247,8 @@ def load_retriever(persist_directory):
         example_no_conversion: If ``False``, the input example will be converted to a Pandas
                 DataFrame format when saving. This is useful when the model expects a DataFrame
                 input and the input example could be passed directly to the model.
-                Defaults to ``True``.
+                Defaults to ``True``. This parameter is deprecated and will be removed in a future
```

Comment:
should this be `None` now?
```suggestion
                Defaults to ``None``. This parameter is deprecated and will be removed in a future
```